### PR TITLE
I had to import StringIO for it to work on my machine? Not sure if my…

### PR DESCRIPTION
**Pull request by kalebhermes**
… computer is just weird.

I'd like to use this in the automation repo, so I added the ability to pass an argument for any RF repo. No argument defaults to Orion.

I added a log message if there are no conflicts.

### CODE REVIEW

File: conflicts.rb [hunk 1]
```diff
@@ -25,8 +26,14 @@ def get_changed_files(change_json)
 current_change_json = JSON.parse(change_details)
 current_changed_files = get_changed_files(current_change_json)
 
-open_changes = StringIO.new `ssh -p 29418 gerrit.rfiserve.net gerrit query --format=JSON --files --current-patch-set status:open project:ui/orion`
+project = 'ui/orion'
+if ARGV[0]
+  project = ARGV[0]
```
Comment (StevenFarberov) [inline conflicts.rb]: I don't think this works if you're using the other command line args such as --recency. I'd recommend making it a --project= option and then accessing via CMD_LINE_ARGS defined above

File: conflicts.rb [hunk 2]
```diff
@@ -25,8 +26,14 @@ def get_changed_files(change_json)
...
+end
+
+open_changes = StringIO.new `ssh -p 29418 gerrit.rfiserve.net gerrit query --format=JSON --files --current-patch-set status:open project:#{project}`
 
+changes = false
```
Comment (StevenFarberov) [inline conflicts.rb]: rename to conflicts?

File: README.md [hunk 3]
```diff
@@ -8,9 +8,19 @@ Run from your git repo:
...
+```
 Output:
 
 ```
 You and Steven Farberov are both modifying the files: src/features/campaign-manage/components/ManageBladeTabs/SomeFile.js
 See Steven Farberov's commit "This is a commit msg" at https://gerrit.rfiserve.net/1234
 ```
+
+### Quick Run
+Add a alias within your bashrc or zshrc config
```
Comment (StevenFarberov) [inline README.md]: 'Add an ... '

File: conflicts.rb [hunk 4]
```diff
@@ -53,3 +61,6 @@ def get_changed_files(change_json)
   end
 end
 
+if changes == false
+    puts "There are no other commits that touch your files."
```
Comment (StevenFarberov) [inline conflicts.rb]: "There are no unmerged commits that also modify your changed files."

**Review by StevenFarberov — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (kalebhermes) [reply README.md]: Updated

File: conflicts.rb (hunk 1 above)
Comment (kalebhermes) [reply conflicts.rb]: Updated

File: conflicts.rb (hunk 2 above)
Comment (kalebhermes) [reply conflicts.rb]: Updated

File: conflicts.rb (hunk 4 above)
Comment (kalebhermes) [reply conflicts.rb]: Updated

File: conflicts.rb [hunk 5]
```diff
@@ -25,8 +26,11 @@ def get_changed_files(change_json)
 current_change_json = JSON.parse(change_details)
 current_changed_files = get_changed_files(current_change_json)
 
-open_changes = StringIO.new `ssh -p 29418 gerrit.rfiserve.net gerrit query --format=JSON --files --current-patch-set status:open project:ui/orion`
+project = CMD_LINE_ARGS['project'] || 'ui/orion'
```
Comment (StevenFarberov) [inline conflicts.rb]: set DEFAULT_PROJECT @ line 8 ?

File: conflicts.rb [hunk 6]
```diff
@@ -53,3 +58,6 @@ def get_changed_files(change_json)
   end
 end
 
+if conflicts == false
+    puts "There are no unmerged commits that also modify your changed files."
```
Comment (StevenFarberov) [inline conflicts.rb]: puts "..." unless conflicts

**Review by StevenFarberov — CHANGES_REQUESTED** [review]

**Review by StevenFarberov — APPROVED** [review]

**Pull request merged**